.github/actions/build_node/entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -l
+
+set -e
+
+if [ -n "$NPM_AUTH_TOKEN" ]; then
+  # Respect NPM_CONFIG_USERCONFIG if it is provided, default to $HOME/.npmrc
+  NPM_CONFIG_USERCONFIG="${NPM_CONFIG_USERCONFIG-"$HOME/.npmrc"}"
+  NPM_REGISTRY_URL="${NPM_REGISTRY_URL-registry.npmjs.org}"
+
+  # Allow registry.npmjs.org to be overridden with an environment variable
+  printf "//%s/:_authToken=%s\\nregistry=%s" "$NPM_REGISTRY_URL" "$NPM_AUTH_TOKEN" "$NPM_REGISTRY_URL" > "$NPM_CONFIG_USERCONFIG"
+  chmod 0600 "$NPM_CONFIG_USERCONFIG"
+fi
+
+PACKAGE_VERSION=$(cat package.json \
```

Comment:
There's a library `jq` for getting values from a JSON file:

```suggestion
PACKAGE_VERSION=$(jq '.version' --raw-output ./package.json)
```

You _might_ need to install it in the Dockerfile, but I think its there.